gulpfile.babel.js

```diff
@@ -139,17 +141,16 @@ gulp.task('bootstrap_test', () => {
 });
 
 gulp.task('build_client', () => {
-  gulp.src(path.resolve(__dirname, 'client', 'index.html'))
+  gulp.src([path.resolve(__dirname, 'client', 'index.html'),
+    path.resolve(__dirname, 'client', 'service-worker.js')])
     .pipe(gulp.dest(path.resolve(__dirname, 'out', 'client')));
   webpack({
     watch: true,
     context: path.resolve(__dirname, 'client'),
-    entry: {
-      bundle: './app.js',
-      sw: './service-worker.js'},
+    entry: './app.js',
     output: {
-      path: path.resolve(__dirname, 'out', 'client'),
-      filename: '[name].js'},
+      path: path.resolve(__dirname, 'out', 'client', 'javascript'),
+      filename: 'bundle.js'},
```

Comment:
- please check duplicated app.js entry
- need to keep [name].js filenama and path to generate right service worker file